src/components/Toast.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect, useCallback, useState, useRef } from 'react';
+import { Text, StyleSheet, View, TouchableOpacity, Animated } from 'react-native';
+
+import { images, icons } from 'app/assets';
+import { Image } from 'app/components';
+import { Toast as IToast } from 'app/consts';
+import { typography, palette } from 'app/styles';
+
+const animationDuration = 500;
+
+interface Props {
+  toast: IToast;
+  onClose: () => void;
+}
+
+export const Toast = ({ toast: { title, description, milisecondsAfterHide }, onClose }: Props) => {
```

Comment:
I don't know about you, but for me, Toast means always mean the native Android component/feature. I'd name it CustomToast, but that's just me :)

Reply:
also, it's quite complicated destructuration here. Maybe it'd be better to do it already in the body?
```
  const { toast: { title, description, milisecondsAfterHide }, onClose } = props;
```

but personally I like the most:
```
  export const Toast = ({ toast, onClose }: Props) => {
```

Reply:
changed to CustomToast